.env.development

```diff
@@ -53,3 +55,4 @@ USE_API_CACHE='true'
 SUBSCRIPTION_LPR='true'
 PLOTLY_SERVER_URL='http://localhost:8050'
 AUTH0_SELF_SERVICE_INTEGRATION='true'
+MFE_CONFIG_API_URL='http://localhost:18000/api/mfe_config/v1'
```

Comment:
`MFE_CONFIG_API_URL` is normally provided by frontends/common/*_config.yml in stage/prod, but in development the only way to supply it currently is via .env.development.